### PR TITLE
gphotos-sync: add recommended package

### PR DESCRIPTION
**Pull request by rhoriguchi**
###### Motivation for this change

Add missing recommended package <https://github.com/gilesknap/gphotos-sync#introduction>:

> the comparison code uses an external tool 'ffprobe'. It will run without it but will not be able to extract metadata from video files and revert to relying on Google Photos meta data and file modified date (this is a much less reliable way to match video files, but the results should be OK if the backup folder was originally created using gphotos-sync).

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by dnr — APPROVED** [review]
Nice, thanks! Did you test the extra functionality?

**Comment by rhoriguchi** [comment]
> Nice, thanks! Did you test the extra functionality?

No, I was setting it up and saw that `ffprobe` is recommended, I'm still waiting for google to approve my application to get API access.

**Comment by dnr** [comment]
It's unfortunate they require approval now. I think when I did it a couple years ago it didn't require any manual steps.

I was just wondering if it would correctly pick up the binary from PATH without any extra work.

Anyway, the change looks good to me but I can't merge myself.

**Pull request merged**
